crates/ekore/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ekore"
+version = "0.1.0"
+edition = "2021"
+description = "EKO expressions"
+license = "GPL-3.0-or-later"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+doctest = false
```

Comment:
@AleCandido I found no other way to make `cargo test` run ... the original error is
```
   Doc-tests ekore
error: proc macro panicked
  --> /home/felix/Physik/N3PDF/EKO/eko/crates/ekore/src/lib.rs:13:5
   |
13 |     bibliothek::include_bibtex!("refs.bib");
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```
the problem seems that `cargo doc` and `cargo test` run in different directories ... any idea?

Reply:
I would try to specify the absolute path with [`env!`](https://doc.rust-lang.org/std/macro.env.html) and [`CARGO_MANIFEST_DIR`](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts) environment variable.

Reply:
I.e. if you invoke:
```rs
env!(`CARGO_MANIFEST_DIR`)
```
it should be replaced at build time with the directory containing the `Cargo.toml` of the package being built, and then you concatenate with `"/refs.bib"` and it should work (I hope - I haven't tested `cargo doc` and `cargo test`, but you will find out immediately).

Reply:
To anticipate you some discoveries about strings:
```rs
env!("CARGO_MANIFEST_DIR").to_owned() + "/refs.bib"
```
in case you never concatenated strings before.

Reply:
```
error: proc macro panicked
  --> ekore/src/lib.rs:12:5
   |
12 |     bibliothek::include_bibtex!((env!("CARGO_MANIFEST_DIR").to_owned() + "/refs.bib"));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: expected path to .bib file as string literal as first parameter: InvalidToken { expected: StringLit, actual: Group, span: Two(#0 bytes(335..388)) }

```

Reply:
Ok, the reason of this is that it is most likely using something like `fmt!()` (or one of its relatives), that actually requires a string literal.

In general, macros (including `macro_rules`, those that look like functions just ending in `!`) are expanded at compiled time (of course), but they might leave the arguments there as they are for some other functions. Instead, `fmt!` manipulates the literal, so it has to be a literal.

Reply:
At this point, I would give up on `bibliothek` completely, and just inline the result of its evaluation (i.e. let's maintain a `.rs` file instead of `.bib` file).

Reply:
let's give the author a chance https://codeberg.org/luleyleo/bibliothek/issues/5 - else we do as you say ...

Reply:
941d64f8